### ISSUE TITLE
Do you still need commit access?

### ISSUE THREAD

**Issue by tstellar** [llvm/llvm-project#91973]
### TLDR: If you want to retain your commit access, please comment on this issue.  Otherwise, you can unsubscribe from this issue and ignore it.  Commit access is not required to contribute to the project.  You can still create Pull Requests without commit access.

@AdamMagierFOSS @DylanFleming-arm @PeixinQiao @piggynl @kavitha-natarajan @ipriyanshi1708 @MarsPLR @Yunzezhu94 @hau-hsu @nirvedhmeshram @mohammed-nurulhoque @realqhc @v1nh1shungry @komalon1 @nhuhuan @tyb-arm @Sophia957 @wjschmidt @el-sc @dpenry

LLVM has a policy of downgrading write access to its repositories for accounts with long term inactivity. This is done because inactive accounts with high levels of access tend to be at increased risk of compromise and this is one tactic that the project employs to guard itself from malicious actors.  Note that write access is not required to contribute to the project.  You can still submit pull requests and have someone else merge them.

Our project policy is to ping anyone with less than five 'interactions' with the repositories over a 12 month period to see if they still need commit access.  An 'interaction' and be any one of:

* Pushing a commit.
* Merging a pull request (either their own or someone else’s).
* Commenting on a PR.

If you want to retain your commit access, please post a comment on this issue.  If you do not want to keep your commit access, you can just ignore this issue.  If you have not responded in 6 weeks, then you will move moved from the 'write' role within the project to the 'triage' role.  The 'triage' role is still a privileged role and will allow you to do the following:

* Review Pull Requests.
* Comment on issues.
* Apply/dismiss labels.
* Close, reopen, and assign all issues and pull requests.
* Apply milestones.
* Mark duplicate issues and pull requests.
* Request pull request reviews.
* Hide anyone’s comments.

**Comment by wjschmidt** [comment]
Please maintain my access.  Thank you!

**Comment by realqhc** [comment]
Please retain my commit access, thanks!

**Comment by mohammed-nurulhoque** [comment]
Please retain my commit access.On May 13, 2024 15:35, Tom Stellard ***@***.***> wrote:
TLDR: If you want to retain your commit access, please comment on this issue.  Otherwise, you can unsubscribe from this issue and ignore it.  Commit access is not required to contribute to the project.  You can still create Pull Requests without commit access.
@AdamMagierFOSS @DylanFleming-arm @PeixinQiao @piggynl @kavitha-natarajan @ipriyanshi1708 @MarsPLR @Yunzezhu94 @hau-hsu @nirvedhmeshram @mohammed-nurulhoque @realqhc @v1nh1shungry @komalon1 @nhuhuan @tyb-arm @Sophia957 @wjschmidt @el-sc @dpenry
LLVM has a policy of downgrading write access to its repositories for accounts with long term inactivity. This is done because inactive accounts with high levels of access tend to be at increased risk of compromise and this is one tactic that the project employs to guard itself from malicious actors.  Note that write access is not required to contribute to the project.  You can still submit pull requests and have someone else merge them.
Our project policy is to ping anyone with less than five 'interactions' with the repositories over a 12 month period to see if they still need commit access.  An 'interaction' and be any one of:
Pushing a commit.Merging a pull request (either their own or someone else’s).Commenting on a PR.
If you want to retain your commit access, please post a comment on this issue.  If you do not want to keep your commit access, you can just ignore this issue.  If you have not responded in 6 weeks, then you will move moved from the 'write' role within the project to the 'triage' role.  The 'triage' role is still a privileged role and will allow you to do the following:
Review Pull Requests.Comment on issues.Apply/dismiss labels.Close, reopen, and assign all issues and pull requests.Apply milestones.Mark duplicate issues and pull requests.Request pull request reviews.Hide anyone’s comments.

**Comment by el-sc** [comment]
Please retain my commit access

________________________________
От: Tom Stellard ***@***.***>
Отправлено: 13 мая 2024 г. 17:35:22
Кому: llvm/llvm-project
Копия: Elena Lepilkina; Mention
Тема: [llvm/llvm-project] Do you still need commit access? (Issue #91973)


TLDR: If you want to retain your commit access, please comment on this issue. Otherwise, you can unsubscribe from this issue and ignore it. Commit access is not required to contribute to the project. You can still create Pull Requests without commit access.

@AdamMagierFOSS<https://github.com/AdamMagierFOSS> @DylanFleming-arm<https://github.com/DylanFleming-arm> @PeixinQiao<https://github.com/PeixinQiao> @piggynl<https://github.com/piggynl> @kavitha-natarajan<https://github.com/kavitha-natarajan> @ipriyanshi1708<https://github.com/ipriyanshi1708> @MarsPLR<https://github.com/MarsPLR> @Yunzezhu94<https://github.com/Yunzezhu94> @hau-hsu<https://github.com/hau-hsu> @nirvedhmeshram<https://github.com/nirvedhmeshram> @mohammed-nurulhoque<https://github.com/mohammed-nurulhoque> @realqhc<https://github.com/realqhc> @v1nh1shungry<https://github.com/v1nh1shungry> @komalon1<https://github.com/komalon1> @nhuhuan<https://github.com/nhuhuan> @tyb-arm<https://github.com/tyb-arm> @Sophia957<https://github.com/Sophia957> @wjschmidt<https://github.com/wjschmidt> @el-sc<https://github.com/el-sc> @dpenry<https://github.com/dpenry>

LLVM has a policy of downgrading write access to its repositories for accounts with long term inactivity. This is done because inactive accounts with high levels of access tend to be at increased risk of compromise and this is one tactic that the project employs to guard itself from malicious actors. Note that write access is not required to contribute to the project. You can still submit pull requests and have someone else merge them.

Our project policy is to ping anyone with less than five 'interactions' with the repositories over a 12 month period to see if they still need commit access. An 'interaction' and be any one of:

  *   Pushing a commit.
  *   Merging a pull request (either their own or someone else’s).
  *   Commenting on a PR.

If you want to retain your commit access, please post a comment on this issue. If you do not want to keep your commit access, you can just ignore this issue. If you have not responded in 6 weeks, then you will move moved from the 'write' role within the project to the 'triage' role. The 'triage' role is still a privileged role and will allow you to do the following:

  *   Review Pull Requests.
  *   Comment on issues.
  *   Apply/dismiss labels.
  *   Close, reopen, and assign all issues and pull requests.
  *   Apply milestones.
  *   Mark duplicate issues and pull requests.
  *   Request pull request reviews.
  *   Hide anyone’s comments.

**Comment by hau-hsu** [comment]
Please retain my commit access, thanks!

**Comment by kavitha-natarajan** [comment]
Please retain my commit access. Thanks!

**Comment by ipriyanshi1708** [comment]
Please retain my commit access. Thanks!

**Comment by komalon1** [comment]
Please maintain my access. Thank you!

**Comment by piggynl** [comment]
Please retain my commit access, thanks!

**Comment by tstellar** [comment]
@AdamMagierFOSS @DylanFleming-arm @PeixinQiao @MarsPLR @Yunzezhu94 @nirvedhmeshram @v1nh1shungry @nhuhuan @tyb-arm @Sophia957 @dpenry Reminder, if you want to retain your commit access, please comment on this ticket in the next week.

**Comment by nirvedhmeshram** [comment]
Please retain my commit access. Thank you.

**Comment by AdamMagierFOSS** [comment]
Please retain my access.All the best,-Adam MagierOn Jun 17, 2024, at 5:46 PM, Tom Stellard ***@***.***> wrote:﻿
@AdamMagierFOSS @DylanFleming-arm @PeixinQiao @MarsPLR @Yunzezhu94 @nirvedhmeshram @v1nh1shungry @nhuhuan @tyb-arm @Sophia957 @dpenry Reminder, if you want to retain your commit access, please comment on this ticket in the next week.

**Comment by Sophia957** [comment]
Please maintain my access. Thank you!

**Comment by Sophia957** [comment]
Please maintain my access. Thank you!



***@***.***
 
From: Tom Stellard
Date: 2024-06-18 06:46
To: llvm/llvm-project
CC: Sophia; Mention
Subject: Re: [llvm/llvm-project] Do you still need commit access? (Issue #91973)
@AdamMagierFOSS @DylanFleming-arm @PeixinQiao @MarsPLR @Yunzezhu94 @nirvedhmeshram @v1nh1shungry @nhuhuan @tyb-arm @Sophia957 @dpenry Reminder, if you want to retain your commit access, please comment on this ticket in the next week.

**Comment by tstellar** [comment]
@DylanFleming-arm @PeixinQiao @MarsPLR @Yunzezhu94 @v1nh1shungry @nhuhuan @tyb-arm @dpenry added to the LLVM Triagers team. If you need write access again, please follow these instructions: https://llvm.org/docs/DeveloperPolicy.html#obtaining-commit-access
